CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [0.13.0] - Unreleased
+## [0.13.0] - 2023-09-06
```

Comment:
@piotr-roslaniec Could you modify L13 for me and change "https://github.com/nucypher/nucypher-core/pull/78" to "https://github.com/nucypher/nucypher-core/pull/80"...whoops!